### ISSUE TITLE
Radio communications lags the entire server

### ISSUE THREAD

**Issue by Razgriz1032** [VOREStation/VOREStation#8454]
#### Brief description of the bug
Any time a message is sent over comms, like on the common channel, including things said by players or station announcements, the game lags for a split second.

#### What you expected to happen
No lag. Normal conversation, emotes, and PDA do not cause this lag.

#### What actually happened
Lag.

#### Steps to reproduce
- Have people talking over the radio and;
- Notice any looping animations (such as scrubbers) jitter for a moment.
- Run back and forth down a hallway and every time a radio message occurs, you stop. Easiest done by ghosting.

#### Code Revision
Server revision: B:release D:
Commit: 4c340c5dac0a7afa8f5f219ec3ba812c5ed67033

**Comment by Razgriz1032** [comment]
Slight update, I did a test today to disable vchat by turning off the core vchat files and such, reverting back to default interface - radio messages still lag the client. This might be due to the recent changes to communications in general.

**Comment by Razgriz1032** [comment]
Another update, I tested it with people talking on supply radio, there is no lag. So it seems to be occurring over common radio.

**Comment by Razgriz1032** [comment]
Tested with a communications blackout even. Seems to still lag when someone tries to communicate over common.

**Comment by Razgriz1032** [comment]
I have set up macros to test in common, a couple departments, and normal chats.

Looking at the Status tab, I see CPU hanging out between 0 ~ 11 when standing around and/or moving. When I press the macro to send a radio message over common, CPU spikes to 35. When I press a macro for a message over a department radio, there is no change. When I press a macro for normal chat, there is no change.

There is a little bit of lag if I spam the normal chat fast enough, and the department chats, but that's because of vchat, and the saved message size is only 150. Setting it to minimum, 50, resolved that lag.

Even when doing a drive-by on an intercom, spamming the normal chat macros, as soon as I get within range of the microphone, it freezes.

I turned off my headset speaker, so I don't see green text, but it still lags.

Anything that produces a message over the common channel causes lag, including ATC, station event announcements, command radio announcements, etc.
